fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -147,6 +147,7 @@ class RocksDBErrorListener : public rocksdb::EventListener {
 };
 using DB = rocksdb::DB*;
 using CF = rocksdb::ColumnFamilyHandle*;
+std::shared_ptr<rocksdb::Cache> rocksdb_block_cache = nullptr;
```

Comment:
Just curious: why don't we use `Reference` here?

Reply:
rocksdb::NewLRUCache() returns std::shared_ptr<rocksdb::Cache>. See: [https://github.com/facebook/rocksdb/blob/fcd32e687b713e172f8a2cec07e30165393fda73/include/rocksdb/cache.h#L120-L126](https://github.com/facebook/rocksdb/blob/fcd32e687b713e172f8a2cec07e30165393fda73/include/rocksdb/cache.h#L120-L126). Is this the answer you are looking for?